app/views/directives/osc-image-summary.html

```diff
@@ -4,10 +4,14 @@
   <div ng-show="resource | annotation:'provider'">Provider: {{ resource | annotation:'provider' }}</div>
   <div ng-show="resource.metadata.namespace">Namespace: {{ resource.metadata.namespace }}</div>
 </div>
-<div class="resource-description"
+<div class="resource-description pad-bottom-none"
      ng-bind-html="resource | description | linky"
```

Comment:
ah we were already running linky on the description... thats really what i was looking for

@luciddreamz do you also want to have the sample repo show up after the description like @MarkDeMaria has here